addon/components/paper-tab-label.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: ''
```

Comment:
This was for public-facing API purposes. I want users to explicitly declare `{{paper-tab-label}}` and `{{paper-tab-body}}` per the `AM` API. The `AM` version doesn't actually render `<md-tab-label>` either.